walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +124,57 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void ConsiderOptionalAttendee() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times.
+    //
+    // Events  :       |--A--|     |--B--|
```

Comment:
i'm a little confused if this comment applies to this test?

Reply:
Comment removed. (Whoops again, need to get in the habit of reviewing my own PRs before requesting reviews).